packages/lib/models/settings/builtInMetadata.ts

```diff
@@ -1553,12 +1553,14 @@ const builtInMetadata = (Setting: typeof SettingType) => {
 		},
 
 		'featureFlag.autoUpdaterServiceEnabled': {
-			value: true,
+			value: false,
 			type: SettingItemType.Bool,
 			public: true,
 			storage: SettingStorage.File,
+			appTypes: [AppType.Desktop],
 			label: () => 'Enable auto-updates',
 			description: () => 'Enable this feature to receive notifications about updates and install them instead of manually downloading them. Restart app to start receiving auto-updates.',
+			show: () => shim.isWindows() || shim.isMac(),
```

Comment:
Auto-updates [seems to be unsupported on Linux](https://github.com/laurent22/joplin/commit/72163018b4a168d8ae0001010cb1dcdb76932303#diff-c8f06d858a0995504e445e3918df266a32acc754589994fcc4bccef75a8e9959R574).